src/organisms/infographic/InfoGraphic.js

```diff
@@ -13,6 +12,7 @@ const InfoGraphic = (props) => {
   } catch (error) {
     console.log(error);
```

Comment:
When we catch error, we have to submit it to Error logger. We use Sentry in the project isn't it? So we have to submit it there. 
It is a good idea to have a function (if Sentry doesn't provide it already) where in the function if ENV=dev it will 
- console.log that error in the console (but not in production ;) )
- and send it to Sentry.